flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/JdbcSinkFunction.java

```diff
@@ -31,9 +31,9 @@
  */
 @Deprecated
 class JdbcSinkFunction extends RichSinkFunction<Row> implements CheckpointedFunction {
```

Comment:
remove this file? I think nobody is using it, and `GenericJdbcSinkFunction` is not accessible for users.